### PR TITLE
Add governing license

### PR DESCRIPTION
**Pull request by SecondFlight**

### CODE REVIEW

File: Main/BasicComponents/GradientBorder.qml [hunk 1]
```diff
@@ -1,3 +1,23 @@
+/*
+    Copyright (C) 2019 Rob van den Berg <rghvdberg at gmail dot org>
```
Comment (grejppi) [inline Main/BasicComponents/GradientBorder.qml]: @rghvdberg probably needs to explicitly agree to the licensing of this file under LGPL

File: Main/Include/snowflake.h [hunk 2]
```diff
@@ -1,3 +1,24 @@
+/*
+    Copyright (C) 2019 Joshua Wade, Henna Haahtti
```
Comment (grejppi) [inline Main/Include/snowflake.h]: *Haahti

File: Main/Include/snowflake.h [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+    Original file by https://github.com/Shenggan/, licensed under MIT
```
Comment (grejppi) [inline Main/Include/snowflake.h]: There's no license header in the original file, but I'd suggest adding the contents of this file into this comment with "Original license below" or something:
https://github.com/Shenggan/SnowFlake/blob/master/LICENSE

File: Main/BasicComponents/GradientBorder.qml (hunk 1 above)
Comment (SecondFlight) [reply Main/BasicComponents/GradientBorder.qml]: Ah yes, thanks for catching that

**Pull request merged**